Dockerfile.dapper

```diff
@@ -66,7 +66,7 @@ RUN ln -s /usr/bin/cni /usr/bin/bridge && \
 
 ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
-ENV GOLANGCI_LINT v1.52.0
+ENV GOLANGCI_LINT v1.56.2
```

Comment:
I noticed the version [here](https://github.com/rancher/rancher/blob/release/v2.8/Dockerfile.dapper#L31) is v1.54.2. Not sure if these need to be in sync but I am assuming not since we don't lint the tests.

Reply:
Yes, they are independent.